src/tests/ftest/util/apricot/apricot/test.py

```diff
@@ -669,6 +669,9 @@ def __init__(self, *args, **kwargs):
         # Suffix to append to each access point name
         self.access_points_suffix = None
 
+        # Option to skip destroying containers during tearDown (default: destroy containers)
+        self.skip_destroy_containers = False
+
```

Comment:
I'd recommend not adding a new `TestWithServers` attribute when this can be handled with `TestContainer` attributes.
```suggestion
```